cpp/src_prims/distance/fused_l2_nn.cuh

```diff
@@ -254,6 +274,35 @@ struct FusedL2NN : public BaseClass {
       }
     }
   }
+
+#if (ENABLE_MEMCPY_ASYNC == 1)
+  ///@todo: fix this to use memcpy_async
```

Comment:
Is this still a relevant comment? Looks like the memcpy is async on line 296.

Reply:
That's a stale comment. Removed.